### PR TITLE
FLUT-879690 - [Others] Added user guidelines for roc and wma indicators in development

### PR DESCRIPTION
**Pull request by natrayansf**
## Description ##

Added user guidelines for ROC and WMA indicators and resolved broken link warnings in the indicators

### CODE REVIEW

File: Flutter/cartesian-charts/technical-indicators.md [hunk 1]
```diff
@@ -654,24 +757,28 @@ Refer the following example below
...
+*	[`Weighted moving average indicator`]() (SMA) - can be rendered with five value mappers ([`xValueMapper`](https://pub.dev/documentation/syncfusion_flutter_charts/latest/charts/TechnicalIndicator/xValueMapper.html), [`lowValueMapper`](https://pub.dev/documentation/syncfusion_flutter_charts/latest/charts/TechnicalIndicator/lowValueMapper.html), [`highValueMapper`](https://pub.dev/documentation/syncfusion_flutter_charts/latest/charts/TechnicalIndicator/highValueMapper.html), [`openValueMapper`](https://pub.dev/documentation/syncfusion_flutter_charts/latest/charts/TechnicalIndicator/openValueMapper.html), [`closeValueMapper`](https://pub.dev/documentation/syncfusion_flutter_charts/latest/charts/TechnicalIndicator/closeValueMapper.html)).
```
Comment (LavanyaGowtham2021) [inline Flutter/cartesian-charts/technical-indicators.md:782]: WMA

**Review by LavanyaGowtham2021 — CHANGES_REQUESTED** [review]

File: Flutter/cartesian-charts/technical-indicators.md [hunk 2]
```diff
@@ -491,10 +493,111 @@ Refer the following example,
...
+            legend: Legend(isVisible: true),
+            indicators: [
+              ROcIndicator<dynamic, dynamic>(
+                period: 3,
+                seriesName: 'HiloOpenClose',
+              ),
+            ],
+            series: <CartesianSeries<ChartData, DateTime>>[
+              HiloOpenCloseSeries<ChartData, DateTime>(
+                name: 'HiloOpenClose',
```
[the remark this answers is not in the document]
Comment (natrayansf) [reply Flutter/cartesian-charts/technical-indicators.md:531]: Added mapper codes

File: Flutter/cartesian-charts/technical-indicators.md (hunk 1 above)
Comment (natrayansf) [reply Flutter/cartesian-charts/technical-indicators.md]: Modified to WMA from SMA.

**Review by LavanyaGowtham2021 — APPROVED** [review]
Fine

**Pull request merged**